docs/parallel_training/recommened_pratice.rst

```diff
@@ -0,0 +1,75 @@
+Recommened Practice
```

Comment:
spelling error: Recommended （filename is also wrong）

Reply:
done